src/guides/v2.3/ui_comp_guide/concepts/ui_comp_uilayout_concept.md

```diff
@@ -114,7 +114,7 @@ The `isTemplate` configuration for the original UI component must be `true` to s
 
 In the following example, a custom Table UI component is created using an existing configuration template with the name `my_row_template_component_name`.
 
-The [`mageUtils`](https://github.com/magento/magento2/tree/2.3-develop/lib/web/mage/utils) helper library is also used in this example to create the new component based on `myRowTemplateConfig`.
+The [`mageUtils`](https://github.com/magento/magento2/tree/2.4-develop/lib/web/mage/utils) helper library is also used in this example to create the new component based on `myRowTemplateConfig`.
```

Comment:
`master` on devdocs should not contain any links to unreleased magento2 codebase.